### ISSUE TITLE
`ignore` returning undefined

### ISSUE THREAD

**Issue by TylerJang27** [tcort/markdown-link-check#305]
# Overview
Running `markdown-link-check -q basic.in.md` with `basic.in.md` contents:

```markdown
[Good link](#test-header)
[Good link](https://www.google.com)
[Bad link](#bad-header)
[Bad link](https://nowhere.com/bad-link)


## test-header

```

outputs exit code 1 and output:
```
/home/tyler/.cache/trunk/tools/markdown-link-check/3.12.0-2ce3b62d69f45d7b221989693fd6ca21/node_modules/markdown-link-check/markdown-link-check:97
                    if (program.opts().ignore.some((ignorePath) => resolved.includes(ignorePath))) {
                                             ^

TypeError: Cannot read properties of undefined (reading 'some')
    at Command.<anonymous> (/home/tyler/.cache/trunk/tools/markdown-link-check/3.12.0-2ce3b62d69f45d7b221989693fd6ca21/node_modules/markdown-link-check/markdown-link-check:97:46)
    at Command.listener [as _actionHandler] (/home/tyler/.cache/trunk/tools/markdown-link-check/3.12.0-2ce3b62d69f45d7b221989693fd6ca21/node_modules/commander/lib/command.js:482:17)
    at /home/tyler/.cache/trunk/tools/markdown-link-check/3.12.0-2ce3b62d69f45d7b221989693fd6ca21/node_modules/commander/lib/command.js:1283:65
    at Command._chainOrCall (/home/tyler/.cache/trunk/tools/markdown-link-check/3.12.0-2ce3b62d69f45d7b221989693fd6ca21/node_modules/commander/lib/command.js:1177:12)
    at Command._parseCommand (/home/tyler/.cache/trunk/tools/markdown-link-check/3.12.0-2ce3b62d69f45d7b221989693fd6ca21/node_modules/commander/lib/command.js:1283:27)
    at Command.parse (/home/tyler/.cache/trunk/tools/markdown-link-check/3.12.0-2ce3b62d69f45d7b221989693fd6ca21/node_modules/commander/lib/command.js:909:10)
    at getInputs (/home/tyler/.cache/trunk/tools/markdown-link-check/3.12.0-2ce3b62d69f45d7b221989693fd6ca21/node_modules/markdown-link-check/markdown-link-check:108:7)
    at main (/home/tyler/.cache/trunk/tools/markdown-link-check/3.12.0-2ce3b62d69f45d7b221989693fd6ca21/node_modules/markdown-link-check/markdown-link-check:252:20)

Node.js v20.11.0
```

# markdown-link-check version
3.12.0 (this does not happen on any earlier versions). The same happens on 3.12.1

# Node.js version
v18.12.1 (the same happens on v20.11.0)

# OS
Ubuntu 20.04.6 LTS

# Notes
This may have a similar root cause to #304, although the symptoms seem different.

**Comment by dklimpel** [comment]
> The same happens on 3.12.1

Is it the same error on 3.12.1?

This line of code `if (program.opts().ignore.some((ignorePath) => resolved.includes(ignorePath))) {`
does not exists anymore in 3.12.1.

**Comment by TylerJang27** [comment]
My mistake. This is in fact fixed in 3.12.1. I was relying on my script that I thought had run on 3.12.1, but turns out it missed the release.
